pkgs/development/tools/gauge/plugins/make-gauge-plugin.nix

```
@@ -0,0 +1,104 @@
+{ stdenvNoCC
+, fetchzip
+, lib
+, writeScript
+}:
+
+{ name
+, data
+, repo
+, releasePrefix
+, systemMap ? null
+, ...
+} @ args:
+let
+  otherArgs = lib.attrsets.removeAttrs args [ "name" "data" "repo" "releasePrefix" "systemMap" ];
+  inherit (stdenvNoCC.hostPlatform) system;
+  isMultiArch = systemMap != null;
+  inherit (if isMultiArch then data.${system} else data) url hash;
+in
+stdenvNoCC.mkDerivation (finalAttrs: (lib.recursiveUpdate {
+  pname = "gauge-plugin-${name}";
```

```suggestion
  pname = "gauge-plugin-${pname}";
```
Maybe less confusing